src/SFA.DAS.AANHub.Application.UnitTests/Apprentices/Commands/CreateApprenticeMemberCommandResponseTest.cs

```diff
@@ -0,0 +1,39 @@
+﻿using FluentAssertions;
+using NUnit.Framework;
+using SFA.DAS.AANHub.Application.Apprentices.Commands;
+using SFA.DAS.AANHub.Domain.Entities;
+
+namespace SFA.DAS.AANHub.Application.UnitTests.Apprentices.Commands
+{
+    [TestFixture]
+    public class CreateApprenticeMemberCommandResponseTest
+    {
+        [Test, AutoMoqData]
+        public void Apprentice_PopulatesMemberFromCreateApprenticeMemberCommand(Member member)
+        {
+            var response = (CreateApprenticeMemberCommandResponse)member;
+
+            Assert.That(response, Is.Not.Null);
+
+            response.Should().BeEquivalentTo(member, m => m
```

Comment:
you don't need both BeEquivalent and the Assert.AreEqual.   Take out one (probably the BeEquivalent)

Reply:
Removed Assert.AreEqual